### PR TITLE
add support to set tls_sni in web service

### PR DESCRIPTION
**Pull request by rohitSangamnerkar**

### CODE REVIEW

File: banyan/resource_service_web.go [hunk 1]
```diff
@@ -486,6 +501,15 @@ func expandBackendWhitelist(d *schema.ResourceData) []string {
 	return items
 }
 
+func expandTLSSNIs(d *schema.ResourceData) []string {
```
Comment (nareshkakubal) [inline banyan/resource_service_web.go:504]: @rohitSangamnerkar default TLSSNI should continue to have "domain" as the default behaviour, if TLSSNI is specified it should be set as is.

File: banyan/resource_service_web.go [hunk 2]
```diff
@@ -286,6 +286,13 @@ func WebSchema() (s map[string]*schema.Schema) {
 			Optional:    true,
 			Description: "access tier group which is associated with service",
 		},
+		"tls_sni": {
```
Comment (nareshkakubal) [inline banyan/resource_service_web.go:289]: @rohitSangamnerkar  add corresponding testcase to cover the scenario

File: banyan/resource_service_web.go (hunk 1 above)
Comment (rohitSangamnerkar) [reply banyan/resource_service_web.go:516]: Updated code use domain as TLSSNI if TLSSNI is not specified. If it is specified existing TLSSNI will be replaced by specified TLSSNI.

File: banyan/resource_service_web.go (hunk 2 above)
Comment (rohitSangamnerkar) [reply banyan/resource_service_web.go:289]: added test cases for both scenarios
1 : using default domain TLSSNI
2 : using custom TLSSNI.

**Review by nareshkakubal — APPROVED** [review]

**Pull request merged**